azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/hangup/LeaveConfirmViewModel.kt

```diff
@@ -23,7 +23,11 @@ internal class LeaveConfirmViewModel(private val store: Store<ReduxState>) {
 
     fun confirm() {
         if (store.getCurrentState().callState.operationStatus == OperationStatus.SKIP_SETUP_SCREEN &&
-            store.getCurrentState().callState.callingStatus != CallingStatus.CONNECTED
+            (
+                store.getCurrentState().callState.callingStatus != CallingStatus.CONNECTED &&
```

Comment:
same as above, can use an array for sets of valid/invalid states, and contains for a bit of readability.

Reply:
As per discussion we will fix for GA.